app/Http/Controllers/ManagerController.php

```diff
@@ -10,6 +10,7 @@
 use Tymon\JWTAuth\Exceptions\JWTException;
 use App\Http\Requests\ManagerStoreRequest;
 
+
```

Comment:
Not sure why there is an added new line here. Accident?

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA